lua/chatgpt.lua

```diff
@@ -37,10 +38,8 @@ M.run_action = function(opts)
   module.run_action(opts)
 end
 
-M.run_custom_code_action = function(opts)
-  module.run_custom_code_action(opts)
-end
-
-M.complete_code = module.complete_code
```

Comment:
Please keep `complete_code `as it is. It's a different flow in the early stage, but I'm going to work on it more to make it work similar to Copilot.